ee/docker-ee/ubuntu.md

```diff
@@ -80,7 +79,6 @@ needs.
   [set up Docker's repositories](#install-using-the-repository) and install
   from them, for ease of installation and upgrade tasks. This is the
   recommended approach.
-
 - Some users download the DEB package and install it manually and manage
   upgrades completely manually. This is useful in situations such as installing
   Docker on air-gapped systems with no access to the internet.
```

Comment:
Under the **Install Docker Engine - Enterprise** heading and before the **Install using the repository** subheading, change all of the text as follows:

Docker Engine - Enterprise can be installed either via Docker repositories, or by downloading and installing the DEB package and thereafter manually managing all upgrades. The Docker repository method is recommended, for the ease it lends in terms of both installation and upgrade tasks. The more manual DEB package approach, however, is useful in certain situations, such as installing Docker on air-gapped system that have no access to the Internet.